synapse/models/inet.py

```diff
@@ -1530,6 +1533,14 @@ def getModelDefs(self):
                         'interfaces': ('inet:service:object',),
                         'doc': 'An account within a service platform. Accounts may be instance specific.'}),
 
+                    ('inet:service:relationship:type:taxonomy', ('taxonomy', {}), {
+                        'interfaces': ('meta:taxonomy',),
+                        'doc': 'A service object relationship type taxonomy.'}),
```

Comment:
Is the taxonomy intended to capture account relationships? If so should we include the word "account" in the definition?